x-pack/plugin/core/src/main/resources/logs-default-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+      },
+      "dynamic_templates": [
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
```

Comment:
There haven't been any problem with 1024 historically so I'm good with keeping it. But I wonder if there is a value for ES which would preferrable?

Reply:
1024 is a good default I think